src/screens/GroceryList.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { View, Text, StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+    container: {
+        flex: 1,
+        backgroundColor: "#fff",
+        alignItems: "center",
+        justifyContent: "center",
+    },
+});
+
+const GroceryList = () => {
```

Comment:
This is not your code, is it? If so make sure to tell the team that you have started creating components.